pkg/tempo/servicegraphprocessor/processor.go

```diff
@@ -261,6 +265,11 @@ func (p *processor) consume(trace pdata.Traces) error {
 	return nil
 }
 
+func spanFailed(span pdata.Span) bool {
+	httpStatusCode, ok := span.Attributes().Get("http.status_code")
+	return span.Status().Code() == pdata.StatusCodeError || (ok && httpStatusCode.IntVal()/100 != 2)
```

Comment:
We should make this configurable. 300s are not errors and 400s are debatable on whether or not you'd want them included as a "failed" request. 

We should also add support for `rpc.grpc.status_code`
 https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#grpc

Reply:
Made configurable, thanks for the suggestion